Chutzpah/TestRunner.cs

```diff
@@ -296,7 +296,19 @@ private bool PerformBatchCompile(ITestMethodRunnerCallback callback, IEnumerable
                                 "Launching test harness '{0}' for file '{1}' in a browser",
                                 testContext.TestHarnessPath,
                                 testContext.FirstInputTestFile);
-                            process.LaunchFileInBrowser(testContext.TestHarnessPath, options.BrowserName);
+
+                            // Allow override from command line.
+                            var browserArgs = testContext.TestFileSettings.BrowserArguments;
+                            if (!string.IsNullOrWhiteSpace(options.BrowserArgs))
+                            {
+                                var path = BrowserPathHelper.GetBrowserPath(options.BrowserName);
+                                browserArgs = new Dictionary<string, string>
+                                {
+                                    { Path.GetFileNameWithoutExtension(path), options.BrowserArgs }

```

Comment:
I am using the configured `BrowserName` to get the path, if it is null the path will be the default path for the computers default browser. With that path, I then get the name of the .exe. So, yes - I am anticipating that it could be null.